spellmonitor.lic

```diff
@@ -110,8 +110,10 @@ spell_action = proc do |server_string|
   end
 
   # The start of one or more active spells!
-  if data =~ %r{^<pushStream id="percWindow"/>}
+  if data =~ %r{<pushStream id="percWindow"/>}
```

Comment:
What is appearing in front of this? Could you share some example data you've collected? Thanks!

Reply:
Well, after thinking about it some more, it doesn't matter what's coming before `<pushStream id="percWindow"/>` because for the parsing of spell data we just want to know what comes immediately _after_ the `<pushStream id="percWindow"/>` tag. I got it now :)

Reply:
I found an example, `<popStream id="combat" /><pushStream id="percWindow"/>Rage of the Clans  (10 roisaen)`